tests/route/test_route_flap.py

```diff
@@ -0,0 +1,159 @@
+import math
+import os
+import yaml
+import requests
+import time
+
+import json
+import logging
+import re
+import ipaddress
+import pytest
+import ptf.testutils as testutils
+import ptf.packet as scapy
+
+from ptf.mask import Mask
+
```

Comment:
This test should be applicable for VS setup. Suggest to add mark 'vs'.

Reply:
OK